File: Python/For_Loop(EXERCISE)/Salary.py
web_sites_opened = int(input())
employee_salary = int(input())

for counter in range(web_sites_opened):

    if employee_salary > 0:
        web_sites = input()

        if web_sites.lower() == "facebook":
            employee_salary -= 150
        if web_sites.lower() == "instagram":
            employee_salary -= 100
        if web_sites.lower() == "reddit":
            employee_salary -= 50

if employee_salary <= 0:
    print("You have lost your salary.")
else:
    print(employee_salary)

File: Java Programing Basics/ExerciseForLoop/src/Salary.java
import java.util.Scanner;


public class Salary {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int n = Integer.parseInt(scanner.nextLine());
        int money = Integer.parseInt(scanner.nextLine());
        int money2 = money;

        for (int i = 1; i <= n; i++) {
            String web = scanner.nextLine();

            if (web.equalsIgnoreCase("Facebook")) {
                money -= 150;
            }

            if (web.equalsIgnoreCase("Instagram")) {
                money -= 100;
            }

            if (web.equalsIgnoreCase("Reddit")) {
                money -= 50;
            } else
                money -= 0;
        }


        if (money <= 0) {
            System.out.println("You have lost your salary.");
        }else
            System.out.println(money);
    }
}
